extensions.json

```diff
@@ -385,6 +385,12 @@
       "id": "dylan-foundry.vscode-dylan",
       "repository": "https://github.com/dylan-lang/vscode-dylan"
     },
+    {
+      "id": "eamodio/vscode-gitlens",
```

Comment:
```suggestion
      "id": "eamodio.vscode-gitlens",
```

Reply:
Done - I also switched version to `10.2.3` that looks, on paper, to the the latest that's likely to work in Theia apps based on recent version of the framework.